src/styles/PieChart.module.css

```diff
@@ -1,6 +1,8 @@
 .pie-chart-container {
+  background-color: #fff;
   border: 0.1rem solid #e0e0e0;
```

Comment:
Are we able to use the component library's color tokens?

Reply:
We are not exporting pure white from `cdc-react`. For all the other colors I updated to use the color tokens (css vars)